### PR TITLE
Added validateaddress endpoint

### PR DESCRIPTION
**Pull request by Kexkey**
Added `validateaddress` endpoint.  Delegates to Bitcoin Core's `validateaddress`.

### CODE REVIEW

File: doc/openapi/v0/cyphernode-api.yaml [hunk 1]
```diff
@@ -1184,6 +1184,38 @@ paths:
...
+      summary: "Get the detailed information about the given address"
+      description: "Returns the detailed information about the given address"
+      operationId: "validateAddress"
+      responses:
+        '200':
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                # TODO: describe response
```
Comment (schulterklopfer) [inline doc/openapi/v0/cyphernode-api.yaml:1208]: Please describe response ;)

**Review by schulterklopfer — CHANGES_REQUESTED** [review]
see comment in openapi document

File: doc/openapi/v0/cyphernode-api.yaml (hunk 1 above)
Comment (Kexkey) [reply doc/openapi/v0/cyphernode-api.yaml]: Done

Comment (schulterklopfer) [reply doc/openapi/v0/cyphernode-api.yaml]: nice :D

**Review by schulterklopfer — APPROVED** [review]
lgtm

**Review by schulterklopfer — APPROVED** [review]

**Pull request merged**